cli/app.go

```diff
@@ -2358,6 +2358,28 @@ Copy multiple files from the machine to a local destination with recursion and k
 							}...),
 							Action: createCommandWithT[machinesPartCopyFilesArgs](MachinesPartCopyFilesAction),
 						},
+						{
+							Name:  "get-ftdc",
+							Usage: "download ftdc data from a machine part",
+							Description: `
+In order to use the get-ftdc command, the machine must have a valid shell type service.
+Organization and location are required flags if using name (rather than ID) for the part.
+Note: There is no progress meter while copying is in progress.
+
+Download FTDC data to the current working directory:
+'viam machine part get-ftdc --organization "org" --location "location" --machine "m1" --part "m1-main"
+
+Download FTDC data to an absolute path:
+'viam machine part get-ftdc --part "m1-main" -r $HOME/Downloads'
```

Comment:
I don't like having these here, it's a little awkward because we have three examples of usage text now (these two plus what's generated in the `UsageText` field) and they're all slightly different. I'd rather that anything special about how this is invoked just be explained in comments and we rely on the single usage text if possible.